### PR TITLE
[13.0][FIX] delivery_postlogistics: several fixes

### PR DESCRIPTION
**Pull request by sebalix**
* fix 'PostLogistics' terms
* make 'Package Code' required for PostLogistics delivery packaging
* show a link listing available package codes for PostLogistics delivery packaging
* update README

Ref. 2275

ping @yvaucher

### CODE REVIEW

File: delivery_postlogistics/postlogistics/web_service.py [hunk 1]
```diff
@@ -190,7 +190,10 @@ def _prepare_attributes(
 
         if not services:
             raise exceptions.UserError(
-                _("No Postlogistics packaging services found in this picking.")
+                _(
+                    "No PostLogistics packaging services found "
+                    "in packaging {} of picking {}."
```
Comment (mmequignon) [inline delivery_postlogistics/postlogistics/web_service.py]:
```suggestion
                    "in packaging {}, for picking {}."
```
I would have written that instead.

**Review by mmequignon — APPROVED** [review]
minor suggestion, LG

**Review by simahawk — APPROVED** [review]

File: delivery_postlogistics/postlogistics/web_service.py (hunk 1 above)
Comment (sebalix) [reply delivery_postlogistics/postlogistics/web_service.py]: fixed

**Pull request merged**